extensions/amp-story/1.0/page-advancement.js

```diff
@@ -482,11 +528,9 @@ class ManualAdvancement extends AdvancementConfig {
    */
   maybePerformNavigation_(event) {
     const target = dev().assertElement(event.target);
+    const pageRect = this.element_./*OK*/getBoundingClientRect();
 
-    if (this.canShowTooltip_(event) &&
-      matches(target, INTERACTIVE_EMBEDDED_COMPONENTS_SELECTORS)) {
-      // Clicked element triggers a tooltip, so we dispatch the corresponding
-      // event and skip navigation.
+    if (this.isHandledByEmbeddedComponent_(event, pageRect)) {
```

Comment:
Any reason why we can't just add the method to check if we clicked on the edges on the screen here? I feel like we're obfuscating how to go through this `if` statement, in a succession of nested methods that each have several conditions :(

Reply:
I did this to improve readability for someone that just wants to know what's the purpose of this `if` statement. Especially because this method `maybePerformNavigation_` is concerned with navigation - and the embedded logic should be secondary. 

If I keep adding several conditions here it might be harder to understand when we actually perform navigation, and when we don't.

If they are more interested in knowing what this specific condition entails, they can always read more by following the method. But by default I feel it should be obfuscated.

If you feel strongly about it then I can just re-add it here.